tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/validators/section.js

```diff
@@ -28,7 +28,7 @@ export const validateAndTestSections = (section, paths, dataMapper) => {
         it(`${section.name} should have every question with correct content`, () => {
             // Conduct self assessment according to path
             cy.get("a.govuk-button.govuk-link").contains("Continue").click();
-            validateAndTestQuestionPages(path, section);
+            navigateAndValidateQuestionPages(path, section, (question, section) => { extraQuestionTests(question, section) });
```

Comment:
nice one! 
you can pass the function directly
```suggestion
            navigateAndValidateQuestionPages(path, section, extraQuestionTests);
```
I'd probably `extraQuestionTests` to `validateQuestionUrlAndHint` to be really explicit and `extraTests` to `validator` for consistency with sections but just minor things

Is there a non-quick navigate recommendations? otherwise I'd drop the `quick` too, as it could mean a lot of different things, I think something like navigation with minimal validation gives more clarity to the naming but thats just how I interpret it so feel free to push back

Reply:
Right, I've done a full renaming and hopefully it makes more sense now and is more consistent.

Reply:
looks good!